src/utils/hooks/useLoadApplication.tsx

```diff
@@ -12,11 +13,13 @@ import {
   ApplicationDetails,
   ApplicationStages,
   TemplateSectionPayload,
+  TemplateType,
   UseGetApplicationProps,
 } from '../types'
 
 const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationProps) => {
   const [application, setApplication] = useState<ApplicationDetails>()
+  const [templateType, setTemplateType] = useState<TemplateType>()
```

Comment:
Maybe call this `templateInfo`? I find `templateType` to imply that it's a Type (as in Typescript), but it isn't.